Bifunctor/Type

```diff
@@ -0,0 +1,11 @@
+  λ(p : Type → Type → Type)
+→ { bimap :
+        ∀(a : Type)
+      → ∀(b : Type)
+      → ∀(c : Type)
+      → ∀(d : Type)
+      → ∀(f : a → b)
+      → ∀(g : c → d)
+      → ∀(pac : p a c)
+      → p b d
```

Comment:
Would prefer this style (which I want to make more consistent throughout dhall-bhat eventually):
```
        ∀(a : Type)
      → ∀(b : Type)
      → ∀(c : Type)
      → ∀(d : Type)
      → (a → b)
      → (c → d)
      → p a c
      → p b d
```